modular_skyrat/modules/loadouts/loadout_items/loadout_datum_shoes.dm

```diff
@@ -46,10 +37,18 @@ GLOBAL_LIST_INIT(loadout_shoes, generate_loadout_items(/datum/loadout_item/shoes
 	name = "Knee Boots"
 	item_path = /obj/item/clothing/shoes/jackboots/knee
 
+/*
+*	MISC BOOTS
+*/
+
 /datum/loadout_item/shoes/timbs
 	name = "Fashionable Boots"
 	item_path = /obj/item/clothing/shoes/jackboots/timbs
 
+/datum/loadout_item/shoes/jungle
+	name = "Brown Jackboots"
```

Comment:
Shouldn't these be under jackboots?

Reply:
I was ehhh on that as they're not actually jackboots. Hm. I should rename them to "Jungle Boots" though if I do that, yes.